compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs

```diff
@@ -104,7 +104,7 @@ featureBigNumeric = Feature
 
 featureExceptions :: Feature
 featureExceptions = Feature
-    { featureName = "DAML Exceptions"
+    { featureName = "Daml Exceptions"
```

Comment:
@remyhaemmerle-da does this string need to remain capitalized as before for some reason? Same with the one below.

Reply:
Does this was added after we change the capitalization ? 
@sofiafaro-da  what do you think ?

Reply:
For what it's worth:

```
$ git grep featureName
compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs:    { featureName :: !T.Text
compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs:    { featureName = "String interning"
compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs:    { featureName = "Type interning"
compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs:    { featureName = "Unstable, experimental features"
compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs:    { featureName = "BigNumeric type"
compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs:    { featureName = "Daml Exceptions"
compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs:    { featureName = "Nat type synonyms"
compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs:    { featureName = "Daml Interfaces"
compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs:    { featureName = "Daml Experimental"
compiler/daml-lf-proto/src/DA/Daml/LF/Proto3/EncodeV1.hs:    | otherwise = error $ "Daml-LF " ++ renderPretty version ++ " cannot encode feature: " ++ T.unpack (featureName feature)
compiler/daml-lf-tools/src/DA/Daml/LF/TypeChecker/Error.hs:      "unsupported feature:" <-> pretty featureName
```

So:

* Other features are not all caps.
* It looks like this field is only used for error reporting towards humans.

So my impression is that this is a safe change, at least given that there's a changelog entry that says error messages have changed (from #13191).

Reply:
This is safe! Thanks for checking!

Reply:
Thanks for lending a couple of extra eyeballs. 🙇🏻‍♂️ 👀 I'll consider this item solved.